Project-I-Social-Card/social-card/src/index.js

```diff
@@ -3,4 +3,10 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
```

Comment:
```suggestion
```

Reply:
You had the right idea, but you should import these into App.js, and layout your components there.